README.md

```diff
@@ -274,3 +305,4 @@ file](LICENSE) for details.
 1. [RTP Payload Format for MPEG-4](https://tools.ietf.org/html/rfc6416)
 1. [Coding of MPEG-4 Audio](https://www.iso.org/standard/42739.html)
 1. [ALSA project library reference](https://www.alsa-project.org/alsa-doc/alsa-lib/index.html)
+1. [Semantic Versioning]
```

Comment:
I don't think that it's required here. I've been trying to keep resources related to the functionality of the project here. Semantic versioning is not specific to bluez-alsa, IMHO all projects should comply with it! :)

Reply:
done.